pom.xml

```diff
@@ -167,6 +193,8 @@
         <antlr4.version>4.7</antlr4.version>
         <developer.java-agent.version>1.0.0-SNAPSHOT</developer.java-agent.version>
         <commons-io.version>2.6</commons-io.version>
+        <identity.inbound.auth.saml.version>5.8.3-SNAPSHOT</identity.inbound.auth.saml.version>
```

Comment:
We do not use SNAPSHOT version of other libraries not in this repository

Reply:
Fixed